kitchen.yml

```diff
@@ -42,9 +40,9 @@ platforms:
   - name: ubuntu-18.04
   - name: ubuntu-20.04
   - name: ubuntu-22.04
-  - name: macos-10.12
+  - name: macos-14
     driver:
-      box: chef/macos-10.12 # private
+      box: chef/macos-14 # private
   - name: macos-latest # Used for CI pipeline
   - name: windows-2016
```

Comment:
Do we need this here?

Reply:
switching it to public vagrant box for macos-14 in new commit